docs/asset_info.md

```diff
@@ -184,82 +196,125 @@ Response:
+                            "type": "block",
+                            "name": "Newcombe House (1-35)"
+                     },
+                     {
+                            "id": "6f22e9ae-3e8a-4e0e-af46-db02eb87f8e6",
+                            "type": "estate",
+                            "name": "Newcombe House Estate Powell Road"
+                     }
+                     ],
+                     "floor": "G",
+                     "totalBlockFloors": "5"
+              },
+              "assetAddress": {
+                     "uprn": "100021065786",
+                     "addressLine1": "Powell Road",
+                     "addressLine2": "Hackney",
+                     "addressLine3": "London",
+                     "addressLine4": "",
+                     "postCode": "E5 8DH",
+                     "postPreamble": "1 Newcome House"
+              },
+              "assetManagement": {
+                     "agent": "HAH",
+                     "areaOfficeName": "Clapton Panel Area Team",
+                     "isCouncilProperty": false,
+                     "managingOrganisation": "LBH, -- Hackney or TMO",
+                     "managingOrganisationId": "6f22e9ae-3e8a-4e0e-af46-db02eb87f8e6",
+                     "owner": "London Borough of Hackney",
+                     "isTMOManaged": true,
+                     "propertyOccupiedStatus" : "Vacant",
+                     "isNoRepairsMaintenance" : true,
+                     "fundingSource" : "TA",
+                     "costCentre" : "D0720",
+                     "councilTaxType" : "House in multiple occupancy (HMO)",
+                     "councilTaxLiability" : "Void only",
+                     "lhaArea" : "Inner East London",
+                     "isTemporaryAccomodation" : true,
+                     "readyToLetDate" : true,
+                     "assetStatus" : "House",
+                     "optionToTax" : true
+              },
+              "assetCharacteristics": {
+                     "numberOfBedRooms": "2",
+                     "numberOfLifts": "1",
+                     "numberOfLivingRooms": "1",
+                     "windowType": "DBL",
+                     "yearConstructed": "1978",
+                     "numberOfShowerRooms" : 4,
+                     "assetPropertyFolderLink" : "https://drive.google.com/drive/folders/1Vo7xUYx-wRNR1NTjE4pJGV07DnYirflU",
+                     "epcExpiryDate" : "31-12-2030",
+                     "fireSafetyCertificateExpiryDate" : "31-12-2030",
+                     "gasSafetyCertificateExpiryDate" : "31-12-2030",
+                     "elecCertificateExpiryDate" : "31-12-2030",
+                     "optionToTax" : true,
+                     "hasStairs" : true,
+                     "numberOfStairs" : 20,
+                     "hasRampAccess" : true,
+                     "hasCommunalAreas" : true,
+                     "hasPrivateBathroom" : true,
+                     "numberOfBathrooms" : 4,
```

Comment:
Is a shower room different from a bathroom in terms of the data we store? My concern is introducing fields that we don't store data for / that don't reflect the data collected.